hack/lib/util.sh

```diff
@@ -244,18 +244,18 @@ kube::util::remove-gen-docs() {
 kube::util::group-version-to-pkg-path() {
   staging_apis=(
   $(
-    pushd ${KUBE_ROOT}/staging/src/k8s.io/api > /dev/null
-      find . -name types.go | xargs -n1 dirname | sed "s|\./||g" | sort
-    popd > /dev/null
-  )
-  )
+    cd ${KUBE_ROOT}/staging/src/k8s.io/api &&
```

Comment:
might as well quote the arg to `cd` while touching this

Reply:
Ok. I think a long-term goal is to be able to build kubernetes, when the source tree is in a directory with a space in it's path (such as `/tmp/foo bar/kubernetes`. Right now it doesn't build for many reasons.